README.odrive.md

```diff
@@ -0,0 +1,52 @@
+# Show how to access a Sharepoint group drive in Office 365 business or education
+
+
+When we click "sync" in a Sharepoint groups Documents page we get an
+"odopen" URL, example:
+
+odopen://sync?userId=97858aa6%2Dcd6a%2D4025%2Da6ee%2D29ba80bcbec5&siteId=%7B0936e7d2%2D1285%2D4c54%2D81f9%2D55e9c8b8613b%7D&webId=%7B345db3b7%2D1ebb%2D484f%2D85e3%2D31c785278051%7D&webTitle=FFCCTT%5FIET&listId=%7BA3913FAC%2D26F8%2D4436%2DA9F6%2DAA6B44776343%7D&listTitle=Documentos&userEmail=georg%2Elehner%40ucan%2Eedu%2Eni&listTemplateTypeId=101&webUrl=https%3A%2F%2Fucanedu%2Esharepoint%2Ecom%2Fsites%2FFFCCTT%5FIET&webLogoUrl=%2Fsites%2FFFCCTT%5FIET%2F%5Fapi%2FGroupService%2FGetGroupImage%3Fid%3D%270e6466dc%2D1d90%2D41b0%2D9c9d%2Df6a818721435%27%26hash%3D636549326262064339&isSiteAdmin=1&webTemplate=64&onPrem=0&scope=OPENLIST"
+
+Urldecoded parameters:
+
+	userId=97858aa6-cd6a-4025-a6ee-29ba80bcbec5
+	siteId={0936e7d2-1285-4c54-81f9-55e9c8b8613b}
+	webId={345db3b7-1ebb-484f-85e3-31c785278051}
+	webTitle=FFCCTT_IET
+	listId={A3913FAC-26F8-4436-A9F6-AA6B44776343}
+	listTitle=Documentos
+	userEmail=georg.lehner@ucan.edu.ni
+	listTemplateTypeId=101
+	webUrl=https://ucanedu.sharepoint.com/sites/FFCCTT_IET
+	webLogoUrl=/sites/FFCCTT_IET/_api/GroupService/GetGroupImage?id='0e6466dc-1d90-41b0-9c9d-f6a818721435'&hash=636549326262064339
+	isSiteAdmin=1
+	webTemplate=64
+	onPrem=0
+	scope=OPENLIST
+
+In the following we refer to these parameters with e.g. $userId,
+wherever there are enclosing braces {} you have to strip them off.
+
+
+1- Get site drive-id with:
```

Comment:
@riversource offers a much simpler way of getting the "drive id" here: https://github.com/skilion/onedrive/issues/283#issuecomment-366899361

(I tried both methods and go the same drive "id").

Specifically, they state:
> I created a Sharepoint website collection called "Backup" in the Sharepoint Admin Center. So the URL is:
> GET https://graph.microsoft.com/v1.0/sites/myname.sharepoint.com:/sites/Backup:/drives

So no need to  go through the pain to find the $siteId, $webId and $listId. All you need is the hostname and the collection (group). Note the placement of colons after hostname and collection name.

Reply:
Thanks for the tip.  My instructions are based on clicking the sync button in sharepoint, which runs  - in my case - `xdg-open` with this long nasty link and items. I have written a Tcl/Tk frontend, which already is able to dissect this URL, extract the driveId and set up the configuration file.
I am just desperately looking for the time to shape it up.  It's called `odmon` and `odopen` and can be found at: http://at.magma-soft.at/cgi-bin/darcsweb.cgi?r=odmon;a=summary